### ISSUE TITLE
Known Issue: TomDroid support

### ISSUE THREAD

**Issue by nerdy-sam** [cweiske/grauphel#5]
I just wanted to leave you some information from the log after an attempt to sync TomDroid and to help with TomDroid support and have an issue to track it's progress. :-)

This is on OC 7.0.2 w/ postgres + php 5.6

```
Error   PHP     Undefined property: OAuthProvider::$verifier at apps/grauphel/lib/oauth.php#88  2014-09-14T19:11:39+00:00
Error   PHP     Argument 1 passed to OCA\Grauphel\Lib\TokenStorage::store() must be an instance of OCA\Grauphel\Lib\OAuth_Token, instance of OCA\Grauphel\Lib\Token given, called in apps/grauphel/controller/oauthcontroller.php on line 178 and defined at apps/grauphel/lib/tokenstorage.php#36  2014-09-14T19:11:38+00:00
Error   PHP Argument 1 passed to OCA\Grauphel\Lib\TokenStorage::store() must be an instance of OCA\Grauphel\Lib\OAuth_Token, instance of OCA\Grauphel\Lib\Token given, called in apps/grauphel/controller/oauthcontroller.php on line 248 and defined at apps/grauphel/lib/tokenstorage.php#36  2014-09-14T19:11:31+00:00
```

**Comment by cweiske** [comment]
See also https://lists.launchpad.net/tomdroid-dev/msg00257.html

**Comment by nerdy-sam** [comment]
Hey! You're supposed to be on vacation. Get back to having fun, we can talk in a few days...

Thanks for being so proactive, contacting tomdroid devs.

Would also like to bring this tool to your attention, might be helpful...
http://requestb.in/

**Comment by cweiske** [comment]
The first part is fixed now: Authorization works with commit 13b040be98. See https://lists.launchpad.net/tomdroid-dev/msg00259.html

Syncing still does not work, but the first step is done.

**Comment by cweiske** [comment]
Tomdroid sync works now with commit 10e03b9b!
